transport_analysis/tests/test_velocityautocorr.py

```diff
@@ -286,10 +427,146 @@ def test_fft_start_stop_step_all_dims(
         # testing the fft algorithm on unit velocity trajectory
         # defined in step_vtraj()
         # test start stop step is working correctly
-        v_simple = VACF(step_vtraj.atoms, dim_type=tdim, fft=True)
-        v_simple.run(start=tstart, stop=tstop, step=tstep)
+        v_fft = VACF(step_vtraj.atoms, dim_type=tdim, fft=True)
+        v_fft.run(start=tstart, stop=tstop, step=tstep)
         poly = characteristic_poly(
             tstop, tdim_factor, first=tstart, step=tstep
         )
         # polynomial must take offset start into account
-        assert_almost_equal(v_simple.results.timeseries, poly, decimal=3)
+        assert_almost_equal(v_fft.results.timeseries, poly, decimal=3)
+
+    @pytest.mark.parametrize(
```

Comment:
You can separate all the tests that use this fixture into a separate class and then make the parameterization over  the class rather than repeat

Reply:
Those tests would include the old ones for VACF too, right?